evm_loader/program/src/account/legacy/legacy_ether.rs

```diff
@@ -0,0 +1,119 @@
+use std::mem::size_of;
+
+use ethnum::U256;
+use solana_program::{account_info::AccountInfo, pubkey::Pubkey};
+
+use crate::{config::STORAGE_ENTRIES_IN_CONTRACT_ACCOUNT, error::Result, types::Address};
+
+pub struct LegacyEtherData {
```

Comment:
If this is marked as deprecated, any code that uses this will cause a compilation error and would require `#[allow(deprecated)]` everywhere. Which is ugly.